third_party/2and3/jinja2/filters.pyi

```diff
@@ -37,10 +37,7 @@ def do_batch(value, linecount, fill_with: Optional[Any] = ...): ...
 def do_round(value, precision: int = ..., method: str = ...): ...
 def do_groupby(environment, value, attribute): ...
 
-class _GroupTuple(tuple):
-    grouper: Any
-    list: Any
-    def __new__(cls, xxx_todo_changeme): ...
+_GroupTuple = NamedTuple("_GroupTuple", [("grouper", Any), ("list", Any)])
```

Comment:
Why not a NamedTuple class?

(Also, I looked it up and at runtime it is just a namedtuple. I don't know where the `xxx_todo_changeme` came from.

Reply:
Using class syntax fails on Python 3.5: https://travis-ci.org/python/typeshed/jobs/591098768. mypy should probably allow it in stubs.